jina/executors/indexers/vector/milvus.py

```diff
@@ -0,0 +1,49 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Tuple
+
+import numpy as np
+
+from . import BaseVectorIndexer
+from .milvusdb.milvusdbhandler import MilvusDBHandler
+
+
+class MilvusIndexer(BaseVectorIndexer):
+
+    def __init__(self, host: str, port: int,
+                 collection_name: str, index_type: str,
+                 index_params: dict, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.host = host
+        self.port = port
+        self.collection_name = collection_name
+        self.index_type = index_type
+        self.index_params = index_params
+
+    def post_init(self):
+        super().post_init()
+        self.milvus = MilvusDBHandler(self.host, self.port, self.collection_name)
+
+    def get_query_handler(self):
+        db_handler = self.milvus.connect()
+        db_handler.build_index(self.index_type, self.index_params)
+        return db_handler
+
+    def get_add_handler(self):
+        return self.milvus.connect()
+
+    def get_create_handler(self):
+        return self.milvus.connect()
+
+    @property
+    def query_handler(self):
+        return self.get_query_handler()
```

Comment:
It's better to make it lazy property.

Reply:
I do not find the documentation on how to check properly if connection is good and whether an index exists, which are two conditions we may want to check to make this behavior lazy. I will raise an issue to https://github.com/milvus-io/pymilvus.

Reply:
https://github.com/milvus-io/pymilvus/issues/236